.github/workflows/flutter-prep.yaml

```diff
@@ -49,6 +49,7 @@ jobs:
         id: flutter-candidate
         run: |
           LATEST_FLUTTER_CANDIDATE=$(./tool/latest_flutter_candidate.sh)
+          LATEST_FLUTTER_CANDIDATE=b9e3aef2390b9b755c3c3cf95f7a418f9d6ada6b
```

Comment:
Can you add a comment and file an issue to remove once the flutter candidate is not broken? Also, to get tests to pass you'll need to merge the changes from https://github.com/flutter/devtools/pull/5779 into this PR